src/patterns/data/default/entity_attribute_location.tsv

```diff
@@ -8209,3 +8209,6 @@ OBA:2050179	growth arrest-specific protein 1  level	PR:000007848	growth arrest-s
 OBA:2050180	hepatoma-derived growth factor-related protein 2 level	PR:000008492	hepatoma-derived growth factor-related protein 2	PATO:0000070	amount	UBERON:0001062	anatomical entity	hepatoma-derived growth factor-related protein 2 amount				https://orcid.org/0000-0001-6757-4744|https://orcid.org/0000-0001-8314-2140	
 OBA:2045290	angiotensin converting enzyme activity attribute	PR:000001322	angiotensin-converting enzyme	PATO:0001414	catalytic activity	UBERON:0001062	anatomical entity	angiotensin converting enzyme catalytic activity trait				https://orcid.org/0000-0001-8314-2140	
 OBA:2045561	2,5-diphenylfuran aggregation in anatomical entity trait	CHEBI:50459	2,5-diphenylfuran	PATO:0001629	aggregated	UBERON:0001062	anatomical entity					https://orcid.org/0000-0001-8314-2140	
+OBA:VT1000161	milk total fat amount	CHEBI:18059	lipid	PATO:0103004	total amount	UBERON:0001913	milk	total milk fat amount				https://orcid.org/0000-0001-8314-2140	
+OBA:2045563	total serum cholesterol amount	CHEBI:16113	cholesterol	PATO:0103004	total amount	UBERON:0001977	blood serum	total serum cholesterol				https://orcid.org/0000-0001-8314-2140	
+OBA:VT0010028	milk total protein amount	CHEBI:36080	protein	PATO:0103004	total amount	UBERON:0001913	milk					https://orcid.org/0000-0001-8314-2140	
```

Comment:
Suggestion: include synonym **"total milk protein amount"** in line with similar previously added terms.